### PR TITLE
[typos] Fix typos across codebase

### PR DESCRIPTION
**Pull request by stavares843**
[typos] Fix typos across codebase

### CODE REVIEW

File: lib/ClangImporter/ClangAdapter.cpp [hunk 1]
```diff
@@ -456,8 +456,8 @@ OmissionTypeName importer::getClangTypeNameForOmission(clang::ASTContext &ctx,
     case clang::BuiltinType::OCLIntelSubgroupAVCImeResult:
     case clang::BuiltinType::OCLIntelSubgroupAVCRefResult:
     case clang::BuiltinType::OCLIntelSubgroupAVCSicResult:
-    case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamout:
-    case clang::BuiltinType::OCLIntelSubgroupAVCImeResultDualRefStreamout:
+    case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamount:
```
Comment (CodaFi) [inline lib/ClangImporter/ClangAdapter.cpp:459]: Please revert these, they're not typos - they refer to "stream out"

File: lib/ClangImporter/ImportType.cpp [hunk 2]
```diff
@@ -360,8 +360,8 @@ namespace {
       case clang::BuiltinType::OCLIntelSubgroupAVCImeResult:
       case clang::BuiltinType::OCLIntelSubgroupAVCRefResult:
       case clang::BuiltinType::OCLIntelSubgroupAVCSicResult:
-      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamout:
-      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultDualRefStreamout:
+      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamount:
```
Comment (CodaFi) [inline lib/ClangImporter/ImportType.cpp:363]: This too

File: lib/IRGen/GenCall.cpp [hunk 3]
```diff
@@ -1102,8 +1102,8 @@ namespace {
       case clang::BuiltinType::OCLIntelSubgroupAVCImeResult:
       case clang::BuiltinType::OCLIntelSubgroupAVCRefResult:
       case clang::BuiltinType::OCLIntelSubgroupAVCSicResult:
-      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamout:
-      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultDualRefStreamout:
+      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultSingleRefStreamount:
+      case clang::BuiltinType::OCLIntelSubgroupAVCImeResultDualRefStreamount:
```
Comment (CodaFi) [inline lib/IRGen/GenCall.cpp:1106]: And here

File: stdlib/private/OSLog/OSLogMessage.swift [hunk 4]
```diff
@@ -192,7 +192,7 @@ public struct OSLogInterpolation : StringInterpolationProtocol {
     isScalar: Bool
   ) -> UInt8 {
     var preamble = self.preamble
-    if privacy.isAtleastPrivate {
+    if privacy.isAt leastPrivate {
```
Comment (CodaFi) [inline stdlib/private/OSLog/OSLogMessage.swift:195]: This got separated

File: test/IDE/complete_attributes.swift [hunk 5]
```diff
@@ -24,9 +24,9 @@ struct MyValue {
 // MEMBER_MyValue-DAG: Decl[StaticVar]/CurrNominal:        val[#Int#];
 // MEMBER_MyValue: End completions
 
-class TestUnkownDanglingAttr1 {
+class TestUnknownDanglingAttr1 {
   @UknownAttr(arg: MyValue.#^ATTRARG_MEMBER^#)
 }
-class TestUnkownDanglingAttr2 {
+class TestUnknownDanglingAttr2 {
```
Comment (CodaFi) [inline test/IDE/complete_attributes.swift:30]: When renaming classes you may need to rename them in fixtures too. Please rerun the test suite.

**Review by CodaFi — CHANGES_REQUESTED** [review]

File: lib/ClangImporter/ClangAdapter.cpp (hunk 1 above)
Comment (stavares843) [reply lib/ClangImporter/ClangAdapter.cpp]: reverted, thanks 🔨

File: lib/ClangImporter/ImportType.cpp (hunk 2 above)
Comment (stavares843) [reply lib/ClangImporter/ImportType.cpp]: reverted, thanks 🔨

File: lib/IRGen/GenCall.cpp (hunk 3 above)
Comment (stavares843) [reply lib/IRGen/GenCall.cpp]: done 🔨

File: stdlib/private/OSLog/OSLogMessage.swift (hunk 4 above)
Comment (stavares843) [reply stdlib/private/OSLog/OSLogMessage.swift]: fixed 🔨

File: lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp [hunk 6]
```diff
@@ -21,7 +21,7 @@
 //    initializer annotated "array.uninitialized_intrinsic". This is done in
 //    `run` method.
 //
-// 2. For every "array.uninintialized_intrinsic" initializer call, try
+// 2. For every "array.unininitialized_intrinsic" initializer call, try
```
Comment (buttaface) [inline lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp:24]: Should be `uninitialized_intrinsic` instead?

File: lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp [hunk 7]
```diff
@@ -368,7 +368,7 @@ void ArrayInfo::getLastDestroys(
   for (SILInstruction *frontierInst : frontier) {
     // Skip frontier instructions at the start of a basic block as they do not
     // follow a destroy_value of the array. Note that the goal is to collect
-    // the last destroys, which must always immediately preceed a frontier
+    // the last destroys, which must always immediately preced a frontier
```
Comment (buttaface) [inline lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp:371]: precede

File: test/SILOptimizer/dont_remove_dynamic_self_arg.swift [hunk 8]
```diff
@@ -12,7 +12,7 @@ func transform<T>(_ a: Any, c: (Any) -> T?) -> T? {
 // CHECK-LABEL: sil {{.*}}@$s4test1XC6testityACSgypFACXDSgypXEfU_ :
 // CHECK: bb0({{.*}}, {{%[0-9]+}} : $@thick @dynamic_self X.Type):
 
-// The checked_cast_addr_br must have %1 as implicit type dependend operand.
+// The checked_cast_addr_br must have %1 as implicit type dependent operand.
```
Comment (buttaface) [inline test/SILOptimizer/dont_remove_dynamic_self_arg.swift:15]: type-dependent

**Review by finagolfin — APPROVED** [review]
Unsure about whether some symbol renaming requires other changes too, hopefully the CI catches those.

File: lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp (hunk 6 above)
Comment (stavares843) [reply lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp]: thanks for reviewing, fixed 🔨

File: lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp (hunk 7 above)
Comment (stavares843) [reply lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp]: fixed 🎉

File: stdlib/public/BackDeployConcurrency/MainActor.swift [hunk 9]
```diff
@@ -46,7 +46,7 @@ import Swift
 extension MainActor {
   /// Execute the given body closure on the main actor.
   ///
-  /// Historical ABI entry point, superceded by the Sendable version that is
+  /// Historical ABI entry point, superseded by the Sendable version that is
```
[the remark this answers is not in the document]
Comment (stavares843) [reply stdlib/public/BackDeployConcurrency/MainActor.swift]: reverted 🔨

File: test/SILOptimizer/dont_remove_dynamic_self_arg.swift (hunk 8 above)
Comment (stavares843) [reply test/SILOptimizer/dont_remove_dynamic_self_arg.swift]: updated 🔨

**Comment by buttaface** [comment]
@artemcm, looks like a noob submitted this, would you run the CI?

**Comment by artemcm** [comment]
@swift-ci test

**Pull request closed without merging**